build/tasks/task-serve.js

```diff
@@ -25,8 +26,14 @@ gulp.task('serve', async () => {
     '--port', ports[0],
   ];
 
+  const runnerArgs = [
+    ...args,
+    ...(yargs.argv.browser ? ['--browser', yargs.argv.browser] : []),
```

Comment:
In my case, I use Safari as my default browser but I'd like to open the frontend in Firefox. The default option works well, but I can easily see scenarios where you'd want to test frontends in different browsers. Copy pasting the same link into multiple places might not always work.